pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -143,6 +151,19 @@ func (r *reportingPlugin) Outcome(outctx ocr3types.OutcomeContext, query types.Q
 			continue
 		}
 
+		countedWorkflowIds := map[string]bool{}
```

Comment:
this `countedWorkflowIds` is redundant. it's hard to see, but the the RegisteredWorkflowIds are derived from `getRegisteredWorkflowsIDs` which only contains unique ids becuase it's derived from a map.

this would be see if `getRegisteredWorkflows` -> `listRegisteredWorkflows` and a comment on that func to say that it's the elements are unique.

Reply:
`countedWorkflowIds` exists for the reason that we need protection from malicious nodes sending a list with one ID repeated multiple times, as @bolekk stated in a previous review. We cannot rely on `Outcome` that what was received is in fact always what `getRegisteredWorkflowsIDs` contains as part of `RegisteredWorkflowIds`.

Reply:
i misread the diff and thought this `obs` was the same as the one on line 128. i was wrong.